src/components/PostsContainer/Post.js

```diff
@@ -1,22 +1,21 @@
 // You will add code in this file
-import React from "react";
+import React, { useState } from "react";
 import CommentSection from "../CommentSection/CommentSectionContainer";
 import LikeSection from "./LikeSection";
 import PostHeader from "./PostHeader";
 
 import "./Posts.css";
+import dummyData from "../../dummy-data";
 
 // pass props in this file to
 const Post = props => {
   // set up state for the likes
-
+  const [likes, setLikes] = useState(dummyData);
```

Comment:
Remember that the initial state for `likes` should be coming from props like `useState(props.post.likes)`